examples/js/fortunewhale.js

```diff
@@ -0,0 +1,168 @@
+const fw = require("fission-workflows");
+
+// Single-task workflow
+module.exports = function () {
+    return fw.task({
+        run: "fortune"
+    })
+};
+
+// Simple dependencies with tasks in the requires
+module.exports = function () {
+    const generateFortune = fw.Task({
+        id: "generateFortune",
+        run: "fortune"
+    });
+
+    const whaleSays = fw.task({
+        run: "whalesay",
+        inputs: {
+            "body": "{ output('generateFortune') }"
```

Comment:
JS-inside-string-inside-JSON is pretty awkward.

Reply:
Agreed, see below for some suggested alternative approaches.